_services/crm.html

```diff
@@ -0,0 +1,101 @@
+---
+layout: services
+name: crm
+title: CRM setup
+icon-bright: "/img/deranged_crm_white.svg"
+icon-dark: "/img/deranged_crm_blue.svg"
+meta: "Expert CRM implementation services for businesses. We partner with HubSpot, Pipedrive, and other leading CRM systems to optimise your sales processes, improve lead conversion, and boost revenue growth. Get certified advice and tailored solutions for your organisation."
+excerpt: "We offer expert CRM implementation services for businesses in partnership with HubSpot & Pipedrive, and other leading CRM systems to help you optimise your sales processes, improve lead conversion, and boost revenue growth. Get certified advice and tailored solutions for your CRM."
+header-img: "img/business-man-walking.jpg"
+top-section: "
+<p>
+  Implementing a Customer Relationship Management (CRM) system is a strategic investment that can significantly impact your business's growth and success. By centralising customer information, enhancing relationships, improving sales management, and providing valuable insights, a CRM positions your business for sustainable growth. 
+</p>
+<p>
+  These systems are extremely powerful and can be complex to implement. We work hand-in-hand with you to help you decide on the best fitting CRM for your organisation and work to implement it for you.
+</p>"
+---
+
+<style>
```

Comment:
Should this have `scoped`? I can't remember where it can be used. Maybe this should actually be in the CSS file instead if it can be used other places down the line.

Reply:
Fixed